elife/scripts/rds-perms2.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
(or rename the other one to use `rds-perms.sh` here, but that's going to be difficult due to dependencies on the existing script I think)